.github/workflows/ci.yml

```diff
@@ -91,6 +91,18 @@ jobs:
     needs: build-ruby
     runs-on: ubuntu-18.04
     services:
+      postgres:
+        image: postgres:latest
+        env:
```

Comment:
If we dropped these credentials ("postgres"/"postgres") and went with the defaults ("postgres"/"password") instead, then I believe you could just eliminate all related env vars from the config (both from the service config and the app configs), assuming they are what gets used when database.yml ends up with blank values.